target/linux/ath79/dts/ar7100_mikrotik_routerboard-rb4xx.dtsi

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+// Based on ar71xx arch/mips/ath79/mach-rb4xx.c
+
+/dts-v1/;
```

Comment:
This belongs to the beginning of the DTS only, so you can remove it here.

Reply:
Ok, will move it.